joblib/parallel.py

```diff
@@ -137,10 +137,7 @@ def __call__(self, *args, **kwargs):
             e_type, e_value, e_tb = sys.exc_info()
             text = format_exc(e_type, e_value, e_tb, context=10,
                               tb_offset=1)
-            if issubclass(e_type, TransportableException):
-                raise
```

Comment:
Why has this been removed? it's better to use `raise` to preserve the original traceback in Python 2.